example/src/components/ComponentDocTemplate.js

```diff
@@ -50,105 +51,119 @@ const TemplateDescriptionText = styled.p`
-    margin: 0px;
-    width: 100%;
-    font-family: Source Code Pro;
-    font-style: normal;
-    font-weight: normal;
-    font-size: 16px;
-    line-height: 24px;
-    color: #000000;
-    /* border: 1px solid orange; */
-`
-
-const TemplatePropsContainer = styled.div`
-    margin: 56px 0px 0px 0px;
-    /* border: 1px solid lightcoral; */
-`
-
-const PropsHeaderText = styled.h3`
-    margin: 0px;
-    font-family: "aktiv-grotesk";
-    font-style: normal;
-    font-weight: normal;
-    font-size: 32px;
-    line-height: 40px;
-    display: flex;
+// const TemplateExampleContainer = styled.div`
+//     margin: 16px 0px 0px 0px;
+//     max-width: 768px;
+//     /* height: 56px; */
+//     padding: 8px 16px 8px 16px;
+//     background: #f4f2f2;
+//     border-left: 4px solid #59af9b;
+//     /* border: 1px solid gold; */
+// `
+
+// const TemplateExampleText = styled.p`
+//     margin: 0px;
+//     width: 100%;
+//     font-family: Source Code Pro;
+//     font-style: normal;
+//     font-weight: normal;
+//     font-size: 16px;
+//     line-height: 24px;
+//     color: #000000;
+//     /* border: 1px solid orange; */
+// `
+
+// const TemplatePropsContainer = styled.div`
+//     margin: 56px 0px 0px 0px;
+//     /* border: 1px solid lightcoral; */
+// `
+
+// const PropsHeaderText = styled.h3`
+//     margin: 0px;
+//     font-family: "aktiv-grotesk";
+//     font-style: normal;
+//     font-weight: normal;
+//     font-size: 32px;
+//     line-height: 40px;
+//     display: flex;
+//     align-items: center;
+//     color: #000000;
+//     /* border: 1px dotted gray; */
+// `
+
+// const PropsInfoContainer = styled.div`
+//     margin: 40px 0px 0px 0px;
+//     /* border: 1px solid lightblue; */
+// `
+
+// const InfoWrapper = styled.div`
+//     margin: 0px 0px 0px 0px;
+//     display: flex;
+//     /* border: 1px solid purple; */
+// `
+
+// const InfoBold = styled.p`
+//     margin: 0px 0px 0px 0px;
+//     width: 96px;
+//     font-family: "aktiv-grotesk";
+//     font-style: normal;
+//     font-weight: medium;
+//     font-size: 20px;
+//     line-height: 24px;
+//     /* display: flex; */
+//     /* align-items: center; */
+//     color: #000000;
+//     /* border: 1px dashed gray; */
+// `
+
+// const InfoRegular = styled.p`
+//     margin: 0px 0px 0px 0px;
+//     width: 100%;
+//     max-height: 56px;
+//     /* width: 150px; */
+//     font-family: "aktiv-grotesk";
+//     font-style: italic;
+//     font-weight: lighter;
+//     font-size: 20px;
+//     line-height: 24px;
+//     /* display: flex; */
+//     /* align-items: center; */
+//     color: #000000;
+//     /* border: 1px dashed red; */
+// `
+
+// const InfoDescription = styled.p`
+//     margin: 0px;
+//     font-family: "aktiv-grotesk";
+//     font-style: normal;
+//     font-weight: 300;
+//     font-size: 20px;
+//     line-height: 24px;
+//     display: flex;
+//     align-items: center;
+//     color: #000000;
+//     /* border: 1px dotted black; */
+// `
```

Comment:
@recondesigns If we no longer need these lines, let's delete them before merging this to master